packages/backend/src/index.ts

```diff
@@ -274,9 +275,44 @@ export const slackService = new SlackService({
     lightdashConfig,
 });
 
+let worker: SchedulerWorker | undefined;
 if (lightdashConfig.scheduler?.enabled) {
-    const worker = new SchedulerWorker({ lightdashConfig });
+    worker = new SchedulerWorker({ lightdashConfig });
     worker.run().catch((e) => {
         Logger.error('Error starting scheduler worker', e);
     });
 }
+
+const onExit = () => {
```

Comment:
Need a single exit handler to gracefull shutdown **both**
- opentelemetry sdk (pushes all metrics in buffer)
- worker (finishes all executing tasks)